resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreaker.java

```diff
@@ -1119,17 +1151,17 @@ EventPublisher onSlowCallRateExceeded(
 
         final private Future<T> future;
         final private OnceConsumer<CircuitBreaker> onceToCircuitbreaker;
-        final private long start;
+        final private Clock clock;
 
         CircuitBreakerFuture(CircuitBreaker circuitBreaker, Future<T> future) {
-            this(circuitBreaker, future, System.nanoTime());
+            this(circuitBreaker, future, Clock.systemUTC());
```

Comment:
it is not better to use Clock.systemDefaultZone() instead ?

Reply:
I saw at few places where we have initialised `Clock` as above:
https://github.com/resilience4j/resilience4j/blob/master/resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/internal/CircuitBreakerStateMachine.java#L94

https://github.com/resilience4j/resilience4j/blob/master/resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/internal/CircuitBreakerStateMachine.java#L126

Hence I kept it the same. Let me know your thoughts.